docs/key-features/features.md

```diff
@@ -2,4 +2,89 @@
 title: Key Features
 ---
 
-TBD
+## Cross-cloud multi-cluster multi-mode management
+The overall relationship between the member cluster and the control plane is shown in the following figure:  
+![overall-relationship.png](../resources/key-features/overall-relationship.png)
+
+* Safe isolation: Create a namespace for each cluster, prefixed with `karmada-es-`.
+* Multi-mode:
+  * Push: Karmada is directly connected to the cluster kube-apiserver.
+  * Pull: Deploy one agent component in the cluster, Karmada delegates tasks to the agent component.
+
+* Multi-cloud support(Only if compliant with Kubernetes specifications)
+  * Support various public cloud vendors.
+  * Support for private cloud.
+  * Support self-built clusters.
+
+## Multi-policy multi-cluster scheduling
+The overall scheduling process is shown in the figure below:  
+![overall-relationship.png](../resources/key-features/overall-scheduling.png)
+
+* Cluster distribution capability under different scheduling strategies
+  * ClusterAffinity: Oriented scheduling based on ClusterName, Label, Field.
+  * Toleration: Scheduling based on Taint and Toleration.
+  * SpreadConstraint: Scheduling based on cluster topology.
+  * ReplicasScheduling: Replication mode and split mode for instanced workloads.
+
+* Differential configuration (OverridePolicy)
+  * ImageOverrider: Differentiated configuration of mirrors.
+  * ArgsOverrider: Differentiated configuration of execution parameters.
+  * CommandOverrider: Differentiated configuration for execution commands.
+  * PlainText: Customized Differentiation Configuration.
+
+The overall rescheduling process is shown in the following figure:  
+![overall-relationship.png](../resources/key-features/overall-rescheduling.png)
```

Comment:
Add a blank line between words and  picture? I think some places look overcrowded on the website.

Reply:
fixed